common/scala/build.gradle

```diff
@@ -89,8 +89,8 @@ dependencies {
     compile "io.reactivex:rxjava:1.3.8"
     compile "io.reactivex:rxjava-reactive-streams:1.2.1"
     compile "com.microsoft.azure:azure-cosmosdb:2.6.2"
+    compile "com.sksamuel.elastic4s:elastic4s-client-esjava_${gradle.scala.depVersion}:7.10.3"
```

Comment:
Is this compatible with elasticsearch 6.x version?

Reply:
I checked that 6.x is not compatible with this PR because 6.x requires a `type` parameter for `index` and `deleteByQuery` method  while 7.x doesn't want it

Reply:
I am concerned about the license.
I didn't look into it deeply but from a certain version of ES-7.x they would no longer support it with the Apache license.
Not sure we can upgrade the ES version.

Reply:
> I didn't look into it deeply but from a certain version of ES-7.x they would no longer support it with the Apache license.
What wouldn't they support? Elastic changed the license model sometime after 7.10.x, so this is still entirely covered Apache 2.0.

It's also worth mentioning we don't _have_ to use Elasticsearch. This update makes it possible to use Opensearch, which is covered entirely by Apache 2.0.

Reply:
> The new scheduler relies on multiple filtered aliases against one ES index. I am unclear ES 7.x support this.

I'm not sure what to tell you, I've been using OpenWhisk with my ES 7.10 (Opensearch) cluster for the last week without any issues...

For what it's worth I'm working on updating the tests so they work correctly.

Reply:
Good point, I don't think I am.

That said, I can't find anything in the Elasticsearch documentation or release notes that suggests "_multiple filtered aliases against one ES index_" would have been removed.

Was it something you read that gave you cause for concern?

Reply:
@style95 I think new scheduler doen't rely on `multiple filtered aliase`, and even not on elasticsearch

we are using `multiple filtered aliase` to organize activations by namespaces in our downstream, we may need to consider changing that

Reply:
Hm..
If we don't use `multiple filtered aliases`, it would create a huge number of indices for all namespaces even if they rarely invoke any actions.
It is necessary to have shared indices for namespaces without less invocation.